### PR TITLE
Explosion Event Block Yield

### PR DESCRIPTION
**Pull request by APickledWalrus**
### Description
This expression adds the ability to get and change the yield (number of blocks dropped) in an explosion event.
I'm not sure this is the proper way to do this, but I wasn't sure how to do a property expression for an event.

---
**Target Minecraft Versions:**     Any
**Requirements:**     No
**Related Issues:**     None

### CODE REVIEW

**Comment by APickledWalrus** [comment]
I fixed some issues and removed RESET, as there isn't really a good number for it to be. I also added in some additional checks for the expression to be more user friendly.

**Comment by APickledWalrus** [comment]
I've updated the naming on this so it is more specific (also to avoid confusion with the other PR's I have opened).

File: src/main/java/ch/njol/skript/expressions/ExprExplosionEventBlockYield.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Explosion Event Block Yield")
+@Description("The percentage of blocks dropped in an explosion event.")
```
Comment (bensku) [inline src/main/java/ch/njol/skript/expressions/ExprExplosionEventBlockYield.java]: percentage of exploded blocks that are dropped?

Comment (APickledWalrus) [reply src/main/java/ch/njol/skript/expressions/ExprExplosionEventBlockYield.java]: Somehow missed this comment! I’ll make a clarity change soon.

**Comment by APickledWalrus** [comment]
Changes made! Sorry for delay.

**Comment by APickledWalrus** [comment]
> 
> 
> If doable, this should also work on the [`BlockExplodeEvent`](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/block/BlockExplodeEvent.html).

Correct me if I'm wrong, but I don't think Skript actually uses/supports the BlockExplodeEvent.

**Comment by APickledWalrus** [comment]
This PR could use some changes. Looking over it now, I don't really like what I chose for the syntax. The changer code could also probably be reworked.

**Pull request merged**